swift/Tests/Keystore/KeyStoreTests.swift

```diff
@@ -87,14 +87,17 @@ class KeyStoreTests: XCTestCase {
 
         try keyStore.update(wallet: wallet, password: "password", newPassword: "testpassword")
 
-        let data = wallet.key.decryptPrivateKey(password: "testpassword")
+        let savedKeyStore = try KeyStore(keyDirectory: keyDirectory)
+        let savedWallet = savedKeyStore.wallets.first(where: { $0.identifier == wallet.identifier })!
```

Comment:
can you conform wallet to `Equatable` ? so you don't need to check `identifier `

Reply:
OK, done.